### PR TITLE
Apply webpack to bundle extension

### PR DESCRIPTION
**Pull request by upgle**
## Description
This PR applies a webpack to bundle this extension.

https://code.visualstudio.com/api/working-with-extensions/bundling-extension

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
 			"request": "launch",
 			"runtimeExecutable": "${execPath}",
 			"args": [
+				"--disable-extensions",
```
Comment (style95) [inline .vscode/launch.json:27]: Is this to disable other extensions other than openwhisk-extension?

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+const path = require('path');
+
+/**@type {import('webpack').Configuration}*/
+const config = {
+    target: 'node', // vscode extensions run in a Node.js-context 📖 -> https://webpack.js.org/configuration/node/
+    entry: './src/extension.ts', // the entry point of this extension, 📖 -> https://webpack.js.org/configuration/entry-context/
+    output: {
+        // the bundle is stored in the 'dist' folder (check package.json), 📖 -> https://webpack.js.org/configuration/output/
+        path: path.resolve(__dirname, 'dist'),
+        filename: 'extension.js',
```
Comment (style95) [inline webpack.config.js:12]: How about using a more detailed name?

Comment (upgle) [reply webpack.config.js:12]: This is the name suggested by VSCode official document.
https://code.visualstudio.com/api/working-with-extensions/bundling-extension#configure-webpack

File: .vscode/launch.json (hunk 1 above)
Comment (upgle) [reply .vscode/launch.json:27]: Yes, I'll add an annotation for this.
https://code.visualstudio.com/api/working-with-extensions/testing-extension#disabling-other-extensions-while-debugging

File: src/test/runTest.ts [hunk 3]
```diff
@@ -27,7 +27,15 @@ async function main() {
         const extensionTestsPath = path.resolve(__dirname, './suite/index');
 
         // Download VS Code, unzip it and run the integration test
-        await runTests({ extensionDevelopmentPath, extensionTestsPath });
+        await runTests({
+            extensionDevelopmentPath,
+            extensionTestsPath,
+            /**
```
Comment (upgle) [inline src/test/runTest.ts:33]: @style95 I've added a description about `--disable-extensions` option here.

File: webpack.config.js (hunk 2 above)
Comment (style95) [reply webpack.config.js:12]: I think that's just a sample rather than a suggested name?

**Review by style95 — APPROVED** [review]
LGTM with a minor nit about the output name.
But it would not block merging this.

File: webpack.config.js (hunk 2 above)
Comment (upgle) [reply webpack.config.js:12]: @style95 
Yes, this is not a suggested name. But many VSCode extensions use extension.js or extension.ts as the entry point(main point), and I think they have been affected by documentation.

I don't have a good idea for the detailed name actually 😂 
Could you recommend the name?

### Ref
- https://github.com/microsoft/vscode-chrome-debug/blob/master/package.json#L78
- https://github.com/DonJayamanne/gitHistoryVSCode/blob/master/webpack.config.js#L72
- https://github.com/eamodio/vscode-gitlens/blob/master/webpack.config.js#L74

Comment (style95) [reply webpack.config.js:12]: How about simply using `openwhisk-extension.js` or `vscode-openwhisk-extension.js`?

**Pull request merged**